public/css/widget/quick-actions.less

```diff
@@ -33,14 +35,46 @@
 }
 
 #layout.twocols:not(.wide-layout) {
-  .quick-actions {
+  .quick-actions .flyout-menu {
     justify-content: space-between;
-    min-width: 100%;
   }
 }
+#layout.minimal-layout .quick-actions {
+  > button {
+    display: block;
+    position: fixed;
+    bottom: 4.5em;
+    right: 1em;
+
+    background: @icinga-blue;
+    .appearance(none);
+    border: none;
+    color: @body-bg-color;
+
+    box-shadow: 0 0 1em 0 rgba(0,0,0,.25);
+
+    .rounded-corners(50%);
+    line-height: 3.5;
+    padding: 0 1em;
+
+    .icon {
+      font-size: 1.5em;
 
-#layout.wide-layout .controls {
-  .quick-actions {
-    float: left;
+      &:before {
+        margin-right: 0;
+      }
+    }
+  }
+
+  .flyout {
+    bottom: 7.5em;
+    top: auto;
+    position: fixed;
+    right: 1em;
+    left: 1em;
+
+    &:before {
+      right: 1.125em;
+    }
```

Comment:
What's that for?

Reply:
It’s to position the flyout's caret correctly.

Reply:
> * Clicking on check now or remove ack nothing happens
It’s been fixed in [Icinga/icingaweb2/feature/redesigned-mobile-menu](https://github.com/Icinga/icingaweb2/tree/feature/redesigned-mobile-menu)

> Please use the *Html methods if possible
✅